lib/migration.js

```diff
@@ -186,7 +186,7 @@ function mixinMigration(MySQL, mysql) {
     function actualize(propName, oldSettings) {
       var newSettings = m.properties[propName];
       if (newSettings && changed(newSettings, oldSettings)) {
-        var pName = self.client.escapeId(propName);
+        var pName = self.columnEscaped(model, propName);
```

Comment:
I want to merge this PR asap, but why do you want to replace documented and widely used `self.columnEscaped` to that.

... ten lines above we use `var colName = expectedColNameForModel(propName, m);` for the same result. 

please, confirm the change and, please, explain the reason